drivers/lora/CMakeLists.txt

```diff
@@ -3,4 +3,5 @@
 zephyr_library_named(loramac-node)
 
 zephyr_library_sources_ifdef(CONFIG_LORA_SHELL shell.c)
+zephyr_library_sources_ifdef(CONFIG_HAS_SEMTECH_RADIO_DRIVERS hal_common.c)
```

Comment:
Better name it as `sx_common.c`? Most of the products from semtech seems to have SX prefix. There is `LLCC68` but that's fine.